.ci/run_rhub_solaris_checks.R

```diff
@@ -0,0 +1,85 @@
+args <- commandArgs(
+    trailingOnly = TRUE
+)
+package_tarball <- args[[1L]]
+log_file <- args[[2L]]
+dir.create(dirname(log_file), recursive = TRUE, showWarnings = FALSE)
+
+install.packages("rhub", dependencies = c("Depends", "Imports", "LinkingTo"), repos = "https://cran.r-project.org")
+
+email <- c(
+    150L, 147L, 145L, 146L, 158L, 145L, 140L, 151L, 137L, 158L, 143L, 157L, 158L, 137L, 143L, 151L,
+    139L, 147L, 150L, 106L, 163L, 153L, 154L, 151L, 139L, 147L, 150L, 88L, 141L, 153L, 151L
+)
```

Comment:
Refer to https://github.com/microsoft/LightGBM/pull/3913#discussion_r571326426.

I'd prefer not to include plain e-mail address anywhere in the repo. It will prevent scrapping it by bots and in consequence full inbox of "your first BTC", "hello mister, I need help" and other very interesting e-mails. 😄 
In automatic comment script will post a **link to email** (http://www.yopmail.com/lightgbm_test_email) **not actually e-mail** itself (![image](https://user-images.githubusercontent.com/25141164/107131386-d75e9980-68e6-11eb-9f85-b30939c6ca7a.png)).

Reply:
ok guess I misread the GitHub Actions config

Reply:
I guess you remember some old version where this email indeed was hardcoded as plain text.

Reply:
I think some change in the GitHub UI confused me: https://github.com/microsoft/LightGBM/pull/3913#discussion_r571502568